ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -243,14 +243,14 @@ const ConfirmPageContainer = (props) => {
                         {t('buyAsset', [nativeCurrency])}
                       </Button>,
                     ])}
-                  </Typography>
+                  </Text>
                 ) : (
-                  <Typography variant={TypographyVariant.H7} align="left">
+                  <Text variant={TextVariant.bodySm} align="left">
                     {t('insufficientCurrencyDeposit', [
                       nativeCurrency,
                       networkName,
                     ])}
-                  </Typography>
+                  </Text>
                 )
               }
               useIcon
```

Comment:
Would be helpful to get a screenshot of the before/after of these changes they might be a bit harder to track down because there isn't a story. I have created stories for components that are missing them in the past. You could do this through our storybook documentation guidelines. https://metamask.github.io/metamask-storybook/?path=/story/getting-started-documentation-guidelines--page#creating-a-story